README.md

```diff
@@ -27,44 +27,20 @@ a variety of nGraph-enabled backends: CPU, GPU, and custom silicon like the
 ## Linux instructions
 
 
-### Option 1: ~Use a pre-built nGraph-TensorFlow bridge~ 
-### :warning: Not Yet vailable for this version 
+### Option 1: Use a pre-built nGraph-TensorFlow bridge 
 
 1. You can install TensorFlow and nGraph using `virtualenv` or in the system Python location. 
 
-2. Install TensorFlow v1.14.0rc0:
+2. Install TensorFlow v1.14.0:
 
-        pip install -U tensorflow==1.14.0rc0
+        pip install -U tensorflow==1.14.0
```

Comment:
Have we tested with 1.14.0? All ci is on `tensorflow==v1.14.0-rc0` right now I think